src/Illuminate/Collections/Arr.php

```diff
@@ -641,6 +641,29 @@ public static function sortRecursive($array, $options = SORT_REGULAR, $descendin
         return $array;
     }
 
+    /**
+     * Conditionally compile classes from an array into a class list.
+     *
+     * @param  array  $array
+     * @return string
+     */
+    public static function toClasses($array)
```

Comment:
If the method does stay here, it `toCssClasses` may be a better name so the context of what the method does is clear when the `Arr` class is used outside Blade.

Reply:
Yeah it was a bit of a struggle to find an appropriate name, `toCssClasses` sounds good but lets see what the Laravel team think 👍

Reply:
Wouldn't `toHtmlClasses` be more appropriate? The point is that this is in the context of view templates, but it's not related to styling necessarily. You could have `@class(['clickable' => $enabled])`to control some JavaScript logic for instance.